### PR TITLE
fix: move MetaProps to its own declaration to allow for overrides

### PR DESCRIPTION
**Pull request by paoloricciuti**
Right now the Meta class doesn't have a Props interface. This make it impossible to add overrides to the props (i'm using interface merging to provide intellisense for some parameters)

With this simple change the users will be able to augment the the typing of the Meta component

### CODE REVIEW

**Review by oscard0m — APPROVED** [review]

**Comment by oscard0m** [comment]
✅ but wait for @JReinhold's review

**Comment by paoloricciuti** [comment]
> FYI the Meta tag will be deprecated soon.

Oh yeah I saw that discussion...well until then 😄 also this could help to keep the typing this way when the meta is moved to the module script

**Review by JReinhold — APPROVED** [review]
LGTM!

**Comment by paoloricciuti** [comment]
Do I need to do something else before merging?

**Comment by shilman** [comment]
:rocket: PR was released in [`v4.0.4`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v4.0.4) :rocket:

**Pull request merged**
